### ISSUE TITLE
Contrast between hl-DiffChange and hl-DiffText on the light theme is too low... right?

### ISSUE THREAD

**Issue by aintyourcat** [savq/melange-nvim#57]
Hello! thanks for the colorscheme, been using it for a while now :D

However, as the title said, i have a little problem regarding the colors of the diff on the light theme, to be more specific with the changed part (did a little research, i think its related with the DiffChange and DiffText highlights, but I'm unsure about it). 

Here is how it looks like when i did :Gvdiffsplit...

![2023-03-25_15-47](https://user-images.githubusercontent.com/66950307/227704632-20a5d71f-7bfb-43d1-92a2-e56abf069eca.png)

... compared with [vscode.nvim](https://github.com/Mofiqul/vscode.nvim).

![2023-03-25_15-48](https://user-images.githubusercontent.com/66950307/227704702-61651011-ca52-4b2a-9eb8-83e26b746db4.png)

**Comment by savq** [comment]
Yes, this could be improved! The saturation could be increated to have more contrast.

It'd also be a good idea to use a different hue for `DiffText`. In dark mode it's not that bad, but in light mode it's really hard to distinguish the light blue from the light purple (because of the [Abney effect](https://en.wikipedia.org/wiki/Abney_effect)).

I'm not sure what would be better. The red in your screenshot has good contrast, but red is usually used for deleted text 🤔 Maybe keep magenta for `DiffChange` and use red just for `DiffText`?
